servers/src/main/java/tachyon/worker/block/BlockWorker.java

```diff
@@ -190,10 +204,26 @@ private TThreadPoolServer createThriftServer() {
   }
 
   /**
+   * Get the worker start time (in UTC) in milliseconds.
+   * @return the worker start time in milliseconds
+   */
+  public long getStartTimeMs() {
+    return mStartTimeMs;
+  }
+
+  /**
+   * Gets the meta data of the entire store.
+   * @return store meta data
+   */
+  public BlockStoreMeta getStoreMeta() {
+    return mBlockDataManager.getStoreMeta();
+  }
+
+  /**
    * Helper method to get the {@link java.net.InetSocketAddress} of the worker.
    * @return the worker's address
    */
-  private InetSocketAddress getWorkerAddress() {
+  public InetSocketAddress getWorkerAddress() {
```

Comment:
Do you mean add the same public method `public InetSocketAddress getWorkerAddress()` in this class? It seems impossible to have one private and another public method with the same signatures. Maybe I can just call the public `getWorkerNetAddress` method to get the worker address and display the host and port on UI.